tests/infra/node.py

```diff
@@ -352,6 +352,8 @@ def _start(
                     )
             except ValueError as ve:
                 LOG.info(f"Failed to parse node certificate file ({pem_path}) : {ve}")
+            else:
+                break
```

Comment:
```suggestion
                    )
                    break
            except ValueError as ve:
                LOG.info(f"Failed to parse node certificate file ({pem_path}) : {ve}")
```

This seems simpler, but equivalent?

Reply:
No preference for me really, but used this one.